src/components/Survey/SurveyFooter.js

```diff
@@ -22,7 +37,17 @@ export const SurveyFooter = () => {
             </Button>
           </div>
           <div className={classNames('body')}>
-            <div></div>
+            <h6>
+              <ContentEditable
+                html={L10ns({
+                  prop: 'endText',
+                  language: language,
+                  l10ns: languageSettings,
+                })}
+                onChange={(e) => handleUpdate({ endText: e.target.value })}
```

Comment:
In any `onChange` handler where you only want the value I would suggest doing this instead:
```suggestion
                onChange={({ target: { value }}) => handleUpdate({ endText: value })}
```
I always abstract this on my generic input field component. It looks like this:
```js
const Input = ({ onChange, update, ...lots_of_other_stuff  }) => (
  <LibraryInputComponent
    onChange={(event) => (
      update ? update(event.target.value) : onChange(event)
    )}
    ... other stuff
  />
)
```
So that if I give it an `update` instead of an `onChange` I just get the value back (but it still works for places where I have not done that, or where something outside my control is using it). 💯